tests/Agent/IntegrationTests/SharedApplications/Common/MultiFunctionApplicationHelpers/NetStandardLibraries/LogInstrumentation/Controllers/TestController.cs

```diff
@@ -37,7 +37,8 @@ public string Get(string logLevel, string message)
                     _logger.LogWarning(message);
                     break;
                 case "ERROR":
-                    _logger.LogError(message);
+                    var exception = ExceptionBuilder.BuildException(message);
+                    _logger.LogError(exception, exception.Message);
```

Comment:
The current implementation does not allow us to test an exception without a message

Reply:
This should be resolved in the new commit.